rust-connectors/sources/http/src/bin/main.rs

```diff
@@ -36,12 +36,18 @@ async fn main() -> Result<()> {
         std::env::set_var("RUST_LOG", "http=info");
     }
 
+    // Error on deprecated metadata options
+    if opts.output_format.is_some() {
+        panic!("ERROR: output_format has been deprecated and renamed as output_parts");
```

Comment:
We should bump the version from `0.1.1` to `0.2.0` as this is a breaking change.

Reply:
thx was thinking about that :) :+1: fixed